serf/config.go

```diff
@@ -246,13 +246,23 @@ type Config struct {
 	// UserEventSizeLimit is maximum byte size limit of user event `name` + `payload` in bytes.
 	// It's optimal to be relatively small, since it's going to be gossiped through the cluster.
 	UserEventSizeLimit int
+
+	// MessageDropper is a callback used for selectively ignoring inbound
+	// gossip messages. This should only be used in unit tests needing careful
+	// control over sequencing of gossip arrival
+	MessageDropper func(typ messageType) bool
```

Comment:
```suggestion
	messageDropper func(typ messageType) bool
```

Making this unexported would prevent it from being used outside of the library and our tests.

Reply:
I thought about this, but it would also make it harder to "copy" the `Config` struct, right?

Reply:
Apparently the thing I thought was a problem isn't.